src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -145,7 +145,7 @@ class DataflowOutputEdge:
     """
 
     state: dace.SDFGState
-    expr: DataExpr
```

Comment:
`result` is probably a better name as `expr`, but I suggest you to make it more consistent with the other two classes especially `MemletInputEdge` and call it `destination`.

Reply:
It is not really a destination, it is rather the sink node of the dataflow. What about `sink`?

Reply:
`sink` has a meaning in DaCe as "node with zero output degree", so it is a bit misleading, how about target?

Reply:
In this case, `expr` is a sink node according to the dace definition. A data node the dataflow writes to.

Reply:
but will it never be read from?

Reply:
The sink node (typically a scalar or a local field) of the dataflow will actually be removed from the SDFG. The edge writing to the sink node will be replaced with an edge passing through the map exit node and writing to a new node, the transient field node (a multi-dimensional array with shape equal to the field operator domain).